### PR TITLE
app-text/sigil: bump v1.2.1

### PR DESCRIPTION
**Pull request by arthurzam**
Simple ebuild version bump (just copy to new version).
Unable to bump to `python3.8` because of `dev-python/css-parser`.

Closes: https://bugs.gentoo.org/719474
Package-Manager: Portage-2.3.99, Repoman-2.3.22

### CODE REVIEW

**Comment by arthurzam** [comment]
Thanks to the [bug](https://bugs.gentoo.org/710390) and @a17r ping, `dev-python/css-parser` has `python3.8` support, therefore I added it to `sigil`.
Fully rebuilt, and worked for some hours inside `sigil` on 3.8 to check for regressions - worked OK.

File: app-text/sigil/sigil-1.2.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	app-text/hunspell:=
+	dev-libs/libpcre:3=[pcre16]
+	$(python_gen_cond_dep \
+		'dev-python/css-parser[${PYTHON_MULTI_USEDEP}]
+		dev-python/lxml[${PYTHON_MULTI_USEDEP}]
+		dev-python/six[${PYTHON_MULTI_USEDEP}]')
```
Comment (juippis) [inline app-text/sigil/sigil-1.2.1.ebuild:28]: I **think** you can now use `PYTHON_USEDEP` here inside `python_gen_cond_dep`. Could you please test and verify it works?

File: app-text/sigil/sigil-1.2.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	>=dev-qt/qtwebengine-5.12:5[widgets]
+	>=dev-qt/qtwidgets-5.12:5
+	sys-libs/zlib[minizip]
+	plugins? ( $(python_gen_cond_dep \
+		'dev-python/chardet[${PYTHON_MULTI_USEDEP}]
+		dev-python/cssselect[${PYTHON_MULTI_USEDEP}]
+		dev-python/cssutils[${PYTHON_MULTI_USEDEP}]
+		dev-python/html5lib[${PYTHON_MULTI_USEDEP}]
+		dev-python/pillow[${PYTHON_MULTI_USEDEP}]
+		dev-python/regex[${PYTHON_MULTI_USEDEP}]') )
```
Comment (juippis) [inline app-text/sigil/sigil-1.2.1.ebuild:42]: Here too. Rest looks good.

File: app-text/sigil/sigil-1.2.1.ebuild (hunk 1 above)
Comment (arthurzam) [reply app-text/sigil/sigil-1.2.1.ebuild]: Thank you. Must have missed the eclass change in `gentoo-dev`. Changed (and tested).

File: app-text/sigil/sigil-1.2.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig
+"
+
+S="${WORKDIR}/Sigil-${PV}"
+
+DOCS=( ChangeLog.txt README.md )
+
+src_configure() {
+	python_export PYTHON_LIBPATH PYTHON_INCLUDEDIR
```
Comment (juippis) [inline app-text/sigil/sigil-1.2.1.ebuild:57]:
```
 * Messages for package app-text/sigil-1.2.1:

 * python_export() is part of private eclass API.
 * Please call python_get*() instead.
```

Comment (arthurzam) [reply app-text/sigil/sigil-1.2.1.ebuild]: Fixed.

**Pull request closed without merging**